src/main/java/org/cryptomator/ui/updatereminder/UpdateReminderComponent.java

```diff
@@ -23,7 +24,10 @@ public interface UpdateReminderComponent {
 	Settings settings();
 
 	default void checkAndShowUpdateReminderWindow() {
-		if (LocalDate.parse(settings().lastUpdateCheck.get()).isBefore(LocalDate.now().minusDays(14)) && !settings().checkForUpdates.getValue()) {
+		var now = Instant.now();
+		var twoWeeksAgo = now.minus(Duration.ofDays(14));
+		if (settings().lastUpdateReminder.get().isBefore(twoWeeksAgo) && !settings().checkForUpdates.getValue() && settings().lastSuccessfulUpdateCheck.get().isBefore(twoWeeksAgo)) {
+			settings().lastUpdateReminder.set(now);
```

Comment:
I'm still not too happy with this change. Let's have a call and discuss this.

Reply:
As discussed, lastUpdateReminder has been completely removed.